server/routes/script.js

```diff
@@ -1,9 +1,59 @@
 const express = require('express');
 const router = express.Router();
+const axios = require('axios');
+
+const url = process.env.api;
 
 router.get('/', (req, res) => {
-    res.render('home');
+    function showMovies(url, page = 1) {
+
+        axios.get(url + page)
+            .then(response => {
+                res.render('home', { appl: response.data.results });
+            }, error => {
+                console.log(error);
+            });
+
+    }
+    showMovies(url);
+
 })
 
+router.post('/search', (req, res) => {
+    const { search } = req.body;
+    const SEARCHAPI = process.env.searchapi;
+
+    if (search) {
+
+        axios.get(SEARCHAPI + search)
+            .then(response => {
+                res.render('home', { appl: response.data.results });
+            }, error => {
+                console.log(error);
+            });
+    }
+});
+let i = 1;
```

Comment:
Please, avoid global scope variables in this manner, if you need to track something over time, use a storage solution like redis, postgres, etc...